CellProfiler.spec

```diff
@@ -67,11 +63,11 @@ exe = EXE(
     pyz,
     a.scripts,
     # options,
-    console=False,
+    console=True,
     debug=False,
     exclude_binaries=True,
-    icon=os.path.join("cellprofiler", "data", "CellProfilerIcon.ico"),
-    name='CellProfiler-App',
+    icon=os.path.join("cellprofiler", "data", "images", "CellProfilerIcon.ico"),
+    name="cp",
```

Comment:
It’s the executable inside the distribution. Because the distribution includes the CellProfiler package (`/cellprofiler`) I cannot use the name `cellprofiler`. In practice this is fine, since the application name uses the name from the bundled distribution (`CellProfiler`).